make/prometheus.yml

```diff
@@ -0,0 +1,33 @@
+# my global config
```

Comment:
Configuration files should not be kept in `make` directory. `make` directory is used to keep make scripts. Please move this file to `etc`.

Besides, remove the comment.

Reply:
Besides, remove all **unused** comments and fields in this file.

Reply:
BTW, `tests/etc` also need one copy of this file.